### PR TITLE
Checks for multiple labels (at least one required)

### LINKED ISSUE #128: run and usage together
Hi,

Fedora container guidelines say: "run or usage | Either provides an Atomic run line, or a human readable example of container execution"

Tests enforce both at the same time.

### LINKED ISSUE #129: do not report missing io.k8s.description if description is present
Do not report missing 'io.k8s.description' if 'description' is present. If one exists, the build system copies it to the other. The same applies for the other way, missing description and present io.k8s.description. No need to have both in Dockerfiles.

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Add checks for the multiple-label checks.
- Allow checking multiple labels, where at least one has to be present.
- Use abstract name for label abstract checks.

-----

Resolves #128, resolves #129

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
As expected, new conu release has broken the tests...

**Comment by lachmanfrantisek** [comment]
Do I have to change the code to the new conu API, or leave it to another PR?

File: tests/integration/test_labels.py [hunk 1]
```diff
@@ -13,10 +13,23 @@
...
+import pytest
 
 import colin
 
 
+@pytest.fixture()
+def empty_ruleset():
+    return {
+        "version": "1",
+        "name": "Laughing out loud ruleset",
```
Comment (TomasTomecek) [inline tests/integration/test_labels.py]: LOL

**Comment by TomasTomecek** [comment]
I'll merge and let's fix in the other PR I just submitted

File: tests/integration/test_labels.py (hunk 1 above)
Comment (lachmanfrantisek) [reply tests/integration/test_labels.py]: That's copied from your test..;-)

**Pull request merged**